packages/shared/lib/core/profile/actions/profiles/checkAndMigrateProfiles.ts

```diff
@@ -331,3 +332,16 @@ function persistedProfileMigrationToV17(existingProfile: IPersistedProfile): voi
     existingProfile.network.chains = newChains
     saveProfile(existingProfile)
 }
+
+/*
+ * Migration 18
+ * Change from 1 testnet to 2: Shimmer Testnet (what we knew as Testnet) & IOTA Testnet.
+ * For this reason, we have to rename the existing testnet network ids to Shimmer Testnet.
+ */
+function persistedProfileMigrationToV18(existingProfile: IPersistedProfile): void {
+    if ((existingProfile.network.id as string) === 'testnet') {
+        existingProfile.network.id = NetworkId.ShimmerTestnet
+        existingProfile.network.name = 'Shimmer Testnet'
```

Comment:
Also add
` existingProfile.network.baseToken = DEFAULT_SHIMMER_TESTNET_BASE_TOKEN`

Reply:
there is no need to add it, the testnet didnt change its base token, its the same as before

Reply:
oh it's true 👌🏼